index.js

```diff
@@ -8,6 +8,8 @@ const weatherWindCmp = document.getElementById("weatherWind");
 const weatherSearchBar = document.getElementById("searchBar");
 const searchBtnCmp = document.getElementById("searchBtn");
 const weatherIconCmp = document.getElementById("weatherIcon");
+const tempUnitCmp = document.getElementById("tempUnit");
+
 
 searchBtnCmp.addEventListener("click", function() {
```

Comment:
You're adding event listener on that line and on line 104. This is not really a correct way how you should do it.
You should always think in terms of functions. Which means that you need to split functionality and have only one addEventListener for the button:
Some thing like that:
```js
searchBtnCmp.addEventListener("click", function() {
  fetchWeatherForCity();
  getAnIconForCity();
}
```
This is not exact code, but I hope you would get the idea. And you need to take care about the errors, because if you can't find a weather for a city => you should not load an image. Because right now it could be a pictures with peoples on them if it's not a city name :)